aiogoogle/client.py

```diff
@@ -185,9 +194,16 @@ async def discover(self, api_name, api_version=None, validate=False):
             api=api_name, version=api_version, validate=False
         )
 
-        discovery_docuemnt = await self.as_anon(request)
+        # Set a custom URL to fetch an API's discovery doc from
+        if api_endpoint_url_fmt:
+            assert isinstance(api_endpoint_url_fmt, str)
```

Comment:
A bit irrelevant, but I'm not a big fan of asserting types outside of tests in Python. It kinda breaks the purpose of using Python as opposed to languages like Typescript for example, Type hints are more Pythonic.